vuepress/docs/next/tutorials/curate/bundle-private-images.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebarDepth: 2
+---
+
+# Install Bundle from a Private Image Repository
+
+This tutorial provides to way to utilize bundles from a private image repository in your Entando projects. Private repositories require user authentication and the steps below use environment variables to pass the Secret for authentication.
+
+**New or Existing Bundle**
+* If you have an existing bundle in a private image repository, start at Step 1: Create the Registry Credentials. 
+* If you are starting a new project, start at [Step 2: Create and Apply the Secret](#step-2-create-and-apply-the-secret).
```

Comment:
If you are... -> For a new project, start
can omit "create and apply..." from link